src/server/req_manager.c

```diff
@@ -1561,17 +1612,27 @@ mgr_sched_set(struct batch_request *preq)
 	int	  bad_attr = 0;
 	svrattrl *plist;
 	int	  rc;
+	pbs_sched *psched;
+
+	psched = find_scheduler(preq->rq_ind.rq_manager.rq_objname);
+	if (!psched) {
+		req_reject(PBSE_UNKSCHED, 0, preq);
+		return;
+	}
+	if (validate_sched_attr_vals(preq, psched))
```

Comment:
There are two places input validation is usually done.  In the ECL layer with the data verify functions and in action functions.  In my mind validation is one of the key uses of the action function.